5/jamie-white-captions.srt

```diff
@@ -649,24 +655,24 @@ because it looks the most like...
 
 154
 00:12:17,840 --> 00:12:22,502
-I don't have application roots yet,
-the roots of where data gets loaded in,
+I don't have an applcation route yet.
```

Comment:
Are you happy with this one @nelstrom? 'application route' could also be ApplicationRoute. The Ember guides used the latter until the introduction of Ember CLI.